installer/pkg/components/components-workspace/OWNERS

```diff
@@ -0,0 +1,9 @@
+
+options:
+  no_parent_owners: true
+
+approvers:
+  - engineering-workspace
+
+labels:
+  - "team: workspace"
```

Comment:
nit: new line. And other files

Reply:
Ordinarily I'd agree, but this is the case throughout the `OWNERS` files - in fact, this is copied directly over from them